packages/backend/src/services/ProjectService/ProjectService.ts

```diff
@@ -85,8 +85,19 @@ import { buildQuery } from '../../queryBuilder';
 import { compileMetricQuery } from '../../queryCompiler';
 import { ProjectAdapter } from '../../types';
 import { runWorkerThread, wrapSentryTransaction } from '../../utils';
+import { VERSION } from '../../version';
 import { hasSpaceAccess } from '../SpaceService/SpaceService';
 
+type RunQueryTags = {
+    projectUuid?: string;
+    userUuid?: string;
+    organizationUuid?: string;
+    chartUuid?: string;
```

Comment:
From reading the issue, what about dashboards? I guess that dashboards contain charts, so that's ok. 
Just wondering because on the issue it mentioned the `content_trigger` 🤔

Reply:
Yes good point, I should have proposed an update to the spec. To answer the questions:

- Dashboards: when we run a query the API doesn't actually know the dashboard context when the query runs! We could pass this from the frontend but I didn't think it was worth the dependency until somebody asks for it explicitly
- content_trigger: we could definitely introduce this but I didn't want to commit to the trigger names yet (without dashboards etc.).

Actually since the issue we should also add scheduled delivery id possible 🤔 

But the idea is to ship this and see what users find useful and we can extend it!

Reply:
Sure @owlas, it makes sense to wait until someone asks for it, and this is a great base for what might come next, until new requirements arise! 
Go for it ✌️